baseplate/lib/prometheus_metrics.py

```diff
@@ -26,316 +26,348 @@
     default_size_start * default_size_factor ** i for i in range(default_size_count)
 ]
 
-# http client labels and metrics
-http_client_latency_labels = [
-    "http_method",
-    "http_success",
-    "http_slug",
-]
-http_client_requests_total_labels = [
-    "http_method",
-    "http_success",
-    "http_response_code",
-    "http_slug",
-]
-http_client_active_requests_labels = [
-    "http_method",
-    "http_slug",
-]
-
-# Latency histogram of HTTP calls made by clients
-# buckets are defined above (from 100µs to ~14.9s)
-http_client_latency_seconds = Histogram(
-    "http_client_latency_seconds",
-    "Latency histogram of HTTP calls made by clients",
-    http_client_latency_labels,
-    buckets=default_latency_buckets,
-)
-
-# Counter counting total HTTP requests started by a given client
-http_client_requests_total = Counter(
-    "http_client_requests_total",
-    "Total number of HTTP requests started by a given client",
-    http_client_requests_total_labels,
-)
-
-# Gauge showing current number of active requests by a given client
-http_client_active_requests = Gauge(
-    "http_client_active_requests",
-    "Number of active requests for a given client",
-    http_client_active_requests_labels,
-)
-
 
 class PrometheusHTTPClientMetrics:
```

Comment:
Very pro moving these into the class! 

Are these also instantiated when the file is loaded, or when the class is first used? Worried about metrics disappearing on restart.

Reply:
they're static variables, they exist even if the class is never instantiated
I guess I don't know for sure if they exist before they're accessed?
but they wouldn't exist without tags provided either way so I don't think we can lose anything here

Reply:
Ah, I didn't realize but of course they need labels first. Thanks!